### PR TITLE
Adding mkdocs changes

### PR DESCRIPTION
**Pull request by ramindu90**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,31 +16,70 @@ The Kafka Sink will create the default partition for a given topic, if the topic
...
+extension you can replace the component <a target="_blank" href="https://github.com/wso2-extensions/siddhi-io-kafka/releases">jar</a> that can be found in the `<STREAM_PROCESSOR_HOME>/lib` directory.
+
+**Using the extension as a <a target="_blank" href="https://wso2.github.io/siddhi/documentation/running-as-a-java-library">java library</a>**
+
+* This extension can be added as a maven dependency along with other Siddhi dependencies to your project.
+
+```
+     <dependency>
+        <groupId>org.wso2.extension.siddhi.io.kafka</groupId>
+        <artifactId>siddhi-io-kafka-parent</artifactId>
```
Comment (suhothayan) [inline README.md]: Not parent, it should be the component

File: README.md [hunk 2]
```diff
@@ -14,31 +16,70 @@ The Kafka Sink will create the default partition for a given topic, if the topic
...
 
-#### Prerequisites for using the feature
- - Download and install Kafka.
- - Then convert and copy the Kafka client jars from the <KAFKA_HOME>/libs directory to the <DAS_HOME>/libs directory
- as follows.
-   - Create a directory (SOURCE_DIRECTORY) in a preferred location in your machine and copy the following JARs to it from the
-   <KAFKA_HOME>/libs directory.
-     - kafka_2.11-0.9.0.1.jar
-     - kafka-clients-0.9.0.1.jar
-     - metrics-core-2.2.0.jar
```
Comment (suhothayan) [inline README.md]: If we need to add any Kafka jars we have to add that to prerequisite.

**Review by suhothayan — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -14,7 +16,19 @@ The Kafka Sink will create the default partition for a given topic, if the topic
...
+* <a target="_blank" href="https://github.com/wso2-extensions/siddhi-io-kafka/issues">Issue tracker</a>
+
+## Latest API Docs
+
+Latest API Docs is <a target="_blank" href="https://wso2-extensions.github.io/siddhi-io-kafka/api/4.0.3-SNAPSHOT">4.0.3-SNAPSHOT</a>.
+
+## How to use
+
+### Prerequisites for using the feature
  - Download and install Kafka.
```
Comment (suhothayan) [inline README.md]: It's not DAS it's SP :p, and have this under how to use in SP section

**Review by suhothayan — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (ramindu90) [reply README.md]: requested change is done in 34e69cc959dc3e1f6c85843569bf68ce4a866bde

**Review by suhothayan — APPROVED** [review]

**Pull request merged**